### DISCUSSION TITLE
[general] Broken reverse depenency with newest R2jags

### DISCUSSION THREAD

**Discussion by giabaio** [wlandau/jagstargets#35] [General]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

@suyusung and I have been working on making some key changes to the current version of `R2jags` (see CHANGELOG [here](https://github.com/suyusung/R2jags/blob/master/CHANGELOG) and the issue [here](https://github.com/suyusung/R2jags/issues/19). 

I think that the new summary table (which displays the DIC penalty using the more appropriate name of `pV` and the revised `jags` function, which allows the user to actually also compute the "proper" `pD`, using the relevant definition) kind of breaks the tests you automatically run on `jagstargets`. 

Would you consider checking and possibly adjusting your package? I think it's mainly in terms of the output you retrieve (and code up in the tests) --- no structural change has happened in `R2jags` (well... that's not really true, as I think we fixed a major bug in making `R2jags` properly use the warmup phase...), but the output is presented differently.

**Comment by wlandau** [comment]
Should be fixed in 392fa9a76dbef3c408648c8da2fcc882baf85c9f
